File: src/main/java/me/diax/diax/util/BotType.java
package me.diax.diax.util;

import lombok.Getter;

import java.awt.Color;

public enum BotType {
    STABLE(Colors.BURPLE),

    PATREON(Colors.CANARY),

    DEV(Colors.GREY);

    public static BotType CURRENT_TYPE = STABLE;

    @Getter
    private final Color mainColor;

    BotType(Color mainColor) {
        this.mainColor = mainColor;
    }
}

File: src/main/kotlin/me/diax/diax/util/style/BotType.kt
package me.diax.diax.util.style

import java.awt.Color

enum class BotType constructor(val mainColor: Color) {
    STABLE(Colors.BURPLE),
    PATREON(Colors.CANARY),
    DEV(Colors.DARK);

    companion object {
        var CURRENT_TYPE = STABLE
    }
}
